### PR TITLE
Playbook to just perform logging cert rotation

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1710424

### CODE REVIEW

**Comment by mburke5678** [comment]
@ewolinetz PTAL

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -2344,47 +2344,28 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
 [[fluentd-redeploy-certs]]
 == Redeploying EFK Certificates
 
-You can redeploy EFK certificates, if needed.
+You can use an Ansible playbook to perform a certificate rotation for the EFK stack without needing to run the install/upgrade playbook.
+
+This playbook deletes the current certificate files, generates new EFK certificates, updates certificate secrets, and restarts components
```
Comment (ewolinetz) [inline install_config/aggregate_logging.adoc]: so, the playbook doesn't restart every component -- just kibana and elasticsearch. Not sure if we want to point that out.

fluentd will reload the certificates upon failing to connect to elasticsearch and a curator pod is created based on the cronjob schedule.

**Comment by ewolinetz** [comment]
question about clarity for readers, otherwise lgtm

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: Thanks for the clarification. I took the DocText in th eBZ too literally, it appears!

**Comment by mburke5678** [comment]
@anpingli Small addition to the docs for an Ansible playbook to redeploy EFK certificates. Please take a look. 
Do you know which versions this applies to? We update back to 3.7.

**Comment by anpingli** [comment]
LGTM,  It is for 3.11 only.

**Pull request merged**